resources/backend.rb

```diff
@@ -30,6 +30,15 @@
 include SensuCookbook::SensuCommonProperties
 
 property :config, Hash, default: { "state-dir": '/var/lib/sensu/sensu-backend' }
+property :username, String, default: 'admin'
+property :password, String, default: 'P@ssw0rd!', sensitive: true
```

Comment:
as mentioned above it will absolutely not have a default and this needs consideration to properly handle.

Reply:
I also removed the default for password.